src/components/ebay-filter-menu-button/component.js

```diff
@@ -7,11 +7,11 @@ module.exports = Object.assign({}, menuUtils, {
         eventUtils.handleEscapeKeydown(originalEvent, () => (this._expander.expanded = false));
     },
 
-    handleMenuChange({ checkedIndex, el, originalEvent }) {
+    handleMenuChange({ checkedIndex, el, originalEvent, index, currentChecked }) {
         // TODO: the event data from the filter-menu should probably
         // change to include which items are checked not just the values.
         this.toggleChecked(checkedIndex);
-        this._emitComponentEvent('change', el, originalEvent);
+        this._emitComponentEvent('change', el, originalEvent, index, currentChecked);
```

Comment:
Maybe anything beyond the 3 "always present" params (i.e. `'change', el, originalEvent`) should be rolled into a single object parameter?  Otherwise this signature could get unwieldy if we keep adding more params.

Reply:
Done, good call